helm/prometheus-rules/templates/alerting-rules/app.rules.yml

```diff
@@ -140,6 +141,40 @@ spec:
         sig: none
         team: honeybadger
         topic: releng
+    # Alert Atlas about EFK deployment failed as detected by app platform.
+    - alert: WorkloadClusterAppFailed
```

Comment:
Can we reuse the alert name? I thought it has to be unique.

Reply:
MIssed that, thanks!